data/strings/sound.txt

```diff
@@ -426,9 +499,135 @@
     tr = çıkın.
     uk = З'їзд.
     vi = Lối thoát.
-    zh-Hans = 出口.
+    zh-Hans = 驶出
     zh-Hant = 出口.
 
+  [onto]
+    comment = Used in dist_direction_onto_street in position 3 unless take_exit_number or onto_exit_number are specified.
+    en = onto
+    ar = إلى
+    be = на
+    cs = na
+    da = ad
+    de = auf
+    el = στην
+    es = hacia
+    es-MX = hacia
+    eu =
+    fa = ﺐﻫ
+    fi = tielle
+    fr = sur
+    hi = पर
+    hr = na
+    hu = a-re
+    id = ke
+    it = sulla
+    ja = 
+    ko =
+    mr = वर
+    nb = inn på
+    nl = naar
+    pl = na
+    pt = para
+    pt-BR = para
+    ro = pe
+    ru = на
+    sk = na
+    sv = in på
+    sw = kwenye
+    th = เข้าสู่
+    tr = yönünde
+    uk = на
+    vi = vào
+    zh-Hans = 上
+    zh-Hant = 上
+
+  [onto_exit_number]
+    comment = TODO: currently unused, consider removing. Used in dist_direction_onto_street instead of "onto" when there's an exit number after. Use "." for none
```

Comment:
Why is it not removed?

Reply:
Was holding it open just in case another language reviewer came along needing it; removing.